src/pages/admin/create-notes.scss

```diff
@@ -0,0 +1,937 @@
+.toolbar-button {
+    color: black;
+    background: white;
+    margin-right: 10px;
+    border-width: 4px;
+    border-radius: 8px;
+    font-size: 14px;
+    width: 110px;
+}
+
+.toolbar-button-container {
+  position: fixed;
+  z-index: 9999;
+}
+
+.DraftEditor-editorContainer {
+    margin-top: 20px;
+    width: 80vw;
+    border: black solid !important;
+    padding-bottom: 20px;
+    padding-left: 5px;
+    padding-right: 5px;
+}
+
+.editor-container {
+    margin-left: 15vw;
+    width: 80vw;
+    padding-bottom: 15vw;
+}
+
+.small-input {
+  width: 30vw;
+}
+
+.big-input {
+  width: 60vw;
+}
+
+.desc-button {
+  position: relative;
+  top: 40px;
+}
+
+.imageURL-output {
+  font-size: small;
+}
+
+.copied-alert {
+  position: fixed;
+  font-size: large;
+  opacity: 0.5;
+  text-align: center;
+}
+
+.tags-button  {
+  color: black;
+  background: white;
+  margin-left: 12px;
+  border-radius: 8px;
+  font-size: 14px;
+  width: 140px;
+}
+
+.tags-item {
+  display: inline;
+}
+
+.tags-item:hover {
+  color: red;
+  cursor: pointer;
+}
+
+.calendar-label {
+  margin-right: 10px;
+}
+
+//below is copied from the source files. some edits made
```

Comment:
If this is another open source library you probably need to include the license text and a link to the project.

Reply:
This is the CSS from react-draft-wysiwyg and I needed to modify some of it. I’ll just include the classes that were actually modified.